src/logger/src/metrics.rs

```diff
@@ -778,8 +798,9 @@ pub struct FirecrackerMetrics {
     pub patch_api_requests: PatchRequestsMetrics,
     /// Metrics related to API PUT requests.
     pub put_api_requests: PutRequestsMetrics,
+    #[cfg(target_arch = "aarch64")]
     /// Metrics related to the RTC device.
-    pub rtc: RTCDeviceMetrics,
+    pub rtc: Arc<RTCDeviceMetrics>,
```

Comment:
why do we require an Arc?

Reply:
Because we METRICS.rtc is moved when we create a new RTC device. RTCDeviceMetrics does not implement Clone trait because SharedIncMetrics can not implement Cloine trait because AtomicUsize does not implement it, so the solution would be to use an arc

Reply:
I see. I just find it a little weird to have an Arc into a static variable. The static will live for the entire duration of the program.

To get rid of this, I would just implement the RTCEvents for a dummy struct, that doesn't contain any members, because we can still access the Metrics global variable.

We can consider this a nit

Reply:
Would be a little weird to increment a dummy metric on rtc implementation and then the real one.

Reply:
There would not be any dummy metric increment. Just the trait implementation would be done on another struct. The function implementations would still use the real metrics, just not from `self`, but by accessing the global metrics object.